drake/solvers/Optimization.h

```diff
@@ -331,70 +331,99 @@ class DRAKEOPTIMIZATION_EXPORT OptimizationProblem {
     return addQuadraticCost(Q, x_desired, variable_views);
   };
 
-  /** addConstraint
-   * @brief adds a constraint to the program.  method specializations ensure
-   * that the constraint gets added in the right way
+  /** addGenericConstraint
+   *
+   * @brief adds a generic constraint to the program.  This should
+   * only be used if a more specific type of constraint is not
+   * available, as it may require the use of a significantly more
+   * expensive solver.
    */
-  void addConstraint(std::shared_ptr<Constraint> con,
+  void addGenericConstraint(std::shared_ptr<Constraint> con,
                      VariableList const& vars) {
```

Comment:
Would it make sense to create a "PR-Staging" fork of Drake that we all can push to and on which we stage and submit our PRs? This is so anyone of us can make minor fixes to the PR and speed up the process by which a PR is merged into master?

As of now, I believe only the original author of a PR can change the PR, which seems inefficient.